src/assets/sass/protocol/components/_button.scss

```diff
@@ -4,36 +4,151 @@
 
 @import '../includes/lib';
 
+/* -------------------------------------------------------------------------- */
+// Primary Button
+
 .mzp-c-button {
     @include border-box;
     @include text-cta;
-    transition: background-color 100ms, border-color 100ms, color 100ms;
+    transition: all 200ms;
     background-color: $color-black;
     border-radius: 2px;
     border: 2px solid $color-black;
-    box-shadow: 0 4px 12px rgba(0, 0, 0, .1);
     color: $color-white;
     cursor: pointer;
     display: inline-block;
     font-weight: bold;
-    padding: $padding-md $padding-lg;
+    padding: calc(#{$padding-md} - 2px) calc(#{$padding-xl} - 2px); //2px extra padding removed to compensate for 2px border.
     text-decoration: none;
 
-    &:hover,
-    &:focus {
-        background-color: $color-white;
+    &:hover {
```

Comment:
Need to retain the `focus` state here for keyboard users.

Reply:
@alexgibson I disagree. We really shouldn't be applying any custom focus states. By removing this selector we are preserving the default focus state. It's a best practices from a usability standpoint to keep the default state because users are already familiar with it and know to look for it when using a site. It's the same reason we don't customize the cursor on our site.

![tabbing](https://user-images.githubusercontent.com/5924575/47465937-cdf7a480-d7a3-11e8-9227-d7307c6c265b.gif)

Reply:
I used to think that too but I was told at the Hack on MDN Accessibility weekend (last month) that really visible focus styles are important. There was also [a great presentation about it](https://noti.st/ericwbailey/TcMJFP/if-it-s-interactive-it-needs-a-focus-style) at a11yToConf. 

I would not mess with the *outline* but dramatic `:focus` states improves the keyboard user experience. (A specific use case for these would be if it was appearing on a blue background.)

Reply:
@michaelmccombie i’m not trying to suggest doing anything with `outline` here. The key thing to remember is that browsers all implement focus rings differently. Whilst browsers like Chrome show a bright blue focus ring like in your screenshot, other browser like Firefox use a smaller dotted black outline. Testing your branch out in Firefox, it’s almost impossible to tell when a button is focused currently.

As @stephaniehobson says, if it’s interactive it should really have a focus style. That can be anything we decide, but don’t mess with the default `outline`.

Reply:
Sounds good to me, thanks Stephanie and Alex for looking in to this. I'll update things 👍